### PR TITLE
Feature/p2022 673 bottom navigation

### PR DESCRIPTION
**Pull request by pawelsanaluta**

### CODE REVIEW

**Comment by dziubinski-intive** [comment]
Do we need these PNGs? Can't we just stick to vectors?

File: app/build.gradle [hunk 1]
```diff
@@ -38,10 +38,11 @@ dependencies {
 
     implementation 'androidx.core:core-ktx:1.7.0'
     implementation 'androidx.appcompat:appcompat:1.3.0'
-    implementation 'com.google.android.material:material:1.4.0'
+    implementation 'com.google.android.material:material:1.5.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
     implementation 'androidx.navigation:navigation-fragment-ktx:2.3.5'
     implementation 'androidx.navigation:navigation-ui-ktx:2.3.5'
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```
Comment (dziubinski-intive) [inline app/build.gradle:45]: do we need this?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -9,4 +9,9 @@
 
     <string name="hello_first_fragment">Hello first fragment</string>
     <string name="hello_second_fragment">Hello second fragment. Arg: %1$s</string>
+
+    <!--    bottom navigation bar item labels   -->
+    <string name="navbar_item1_label">Home</string>
```
Comment (dziubinski-intive) [inline app/src/main/res/values/strings.xml:14]: please rename what's behind, it should be:

<string name="home">Home</string>
<string name="search">Search</string>
<string name="favorites">Favorites</string>

**Comment by dziubinski-intive** [comment]
> Do we need these PNGs? Can't we just stick to vectors?

Looks like you just put it into mipmap by mistake, think you can simple remove it.

**Comment by Grafi01234** [comment]
That's a lot of file changes, it's hard to keep track of all of them. 

- I would suggest adding things that have been added/changed/removed as headers to the pull request description for clarity. 
- Additionally, @ Rafał Depta suggested to me adding screenshots before and after from the emulator for visualization of changes, which seems to be a good idea.
- A link from Grzegorz regarding the icons directed me to **vector assets studio** https://developer.android.com/studio/write/vector-asset-studio. This is a different method of adding icons, but it was simpler for me.

**Comment by dziubinski-intive** [comment]
please resolve conflicts

**Pull request merged**